lib/fpm/package/python.rb

```diff
@@ -322,9 +514,57 @@ def fix_name(name)
     end
   end # def fix_name
 
-  # Install this package to the staging directory
-  def install_to_staging(setup_py)
-    project_dir = File.dirname(setup_py)
+
+  # Install this package to the staging directory via pyproject.toml
+  def install_to_staging_toml(wheel_path)
+    project_dir = File.dirname(wheel_path)
+
+    prefix = "/"
+    prefix = attributes[:prefix] unless attributes[:prefix].nil?
+
+    # Some setup's assume $PWD == current directory of pyproject.toml, so let's
+    # chdir first.
+    ::Dir.chdir(project_dir) do
+      flags = [ "--root", staging_path ]
+
+      # if !attributes[:python_install_lib].nil?
```

Comment:
Lot's of commented code. Is it necessary?

Reply:
Seems there is just no alternative for such option (--prefix) in setup.py for PEP517 builds...